django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -0,0 +1,985 @@
+/* Tablets */
+
+input[type="submit"], button {
+    -webkit-appearance: none;
+    appearance: none;
+}
```

Comment:
`appearance` stuff is outside of media query because it's now (in 2017 with a lot of devices) difficult to detect tablet by display width. So by default this rule works globally but it's placed in `responsive.css` because it affects mobile devices only.